src/spandrel/architectures/DAT/__init__.py

```diff
@@ -1,135 +1,90 @@
 import math
-import re
 
 from ...__helpers.model_descriptor import SizeRequirements, SRModelDescriptor, StateDict
+from ..__arch_helpers.state import get_seq_len
 from .arch.DAT import DAT
 
 
 def load(state_dict: StateDict) -> SRModelDescriptor[DAT]:
     # defaults
-    img_size = 64
+    img_size = 64  # cannot be deduced from state dict in general
     in_chans = 3
     embed_dim = 180
-    split_size = [2, 4]
+    split_size = [2, 4]  # very complicated to deduce in general
```

Comment:
it's not complicated to deuce, it just affects an actual weight value rather than a shape, so it's just different

Reply:
> it's not complicated to deuce

On the account that the old code detects `split_size` incorrectly in many cases (just not for default value), I'm going to have to disagree with your assessment. Or put more succinctly: nah, shit's complicated.

Reply:
Nope, my bad. I was stupid.